### PR TITLE
add footer

### PR DESCRIPTION
**Pull request by dikhomenko**

### CODE REVIEW

File: src/css/footer.css [hunk 1]
```diff
@@ -1,3 +1,92 @@
...
-  /* styles */
+* {
+  box-sizing: border-box;
 }
+
+body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+}
```
Comment (OleinikovArtem) [inline src/css/footer.css]: remove

File: src/css/footer.css [hunk 2]
```diff
@@ -1,3 +1,92 @@
-.footer {
-  /* styles */
+* {
+  box-sizing: border-box;
 }
```
Comment (OleinikovArtem) [inline src/css/footer.css]: remove

File: src/partials/footer.html [hunk 3]
```diff
@@ -1,15 +1,82 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/footer.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/izitoast/1.4.0/css/iziToast.min.css"
+    />
+    <title>Footer</title>
+  </head>
+  <body>
```
Comment (OleinikovArtem) [inline src/partials/footer.html]: there are should be only `<footer>...</footer>`; Please remove all stuff around.

File: src/partials/footer.html [hunk 4]
```diff
@@ -1,15 +1,82 @@
...
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/footer.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/izitoast/1.4.0/css/iziToast.min.css"
+    />
```
Comment (OleinikovArtem) [inline src/partials/footer.html]: Connect it to the main index.html \ page-2.html

File: src/footer.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
+document.getElementById("send-button").addEventListener("click", submitForm);
+document.getElementById("email").addEventListener("keydown", function (event) {
+  if (event.key === "Enter") {
+    submitForm(); // Call the submitForm function if Enter key is pressed
+  }
+});
```
Comment (OleinikovArtem) [inline src/footer.js:6]: please make const like `const sendBtn = doc...` and use it

File: src/css/footer.css [hunk 6]
```diff
@@ -1,3 +1,92 @@
...
+  margin-bottom: 20px;
+}
+
+.logo {
+  font-size: 24px;
+  font-weight: bold;
+  cursor: pointer;
+}
+
+.social-links {
```
Comment (OleinikovArtem) [inline src/css/footer.css:19]: we also have these links in header, so or make it more specific or make different component and use it here

File: src/css/footer.css (hunk 1 above)
Comment (dikhomenko) [reply src/css/footer.css]: done

File: src/css/footer.css (hunk 2 above)
Comment (dikhomenko) [reply src/css/footer.css]: done

File: src/partials/footer.html (hunk 3 above)
Comment (dikhomenko) [reply src/partials/footer.html]: done

File: src/partials/footer.html (hunk 4 above)
Comment (dikhomenko) [reply src/partials/footer.html:10]: done

Comment (dikhomenko) [reply src/partials/footer.html]: done

**Pull request merged**